src/components/UI/Button/Button.test.js

```diff
@@ -18,7 +18,10 @@ describe("Button component", () => {
 
   it("shows button text", () => {
     const button = renderer.create(<Button>Test Button</Button>);
-    expect(button.toJSON()).toMatchSnapshot();
+    expect(button.toJSON().children[0]).toMatch("Test Button");
+    expect(button.toJSON().props.className).toMatch(
```

Comment:
let's say we dont care about this :) (to remove)

Reply:
Removed the test